### ISSUE TITLE
Run pack obfuscated scripts got error

### ISSUE THREAD

**Issue by openesa** [dashingsoft/pyarmor#68]
Problem: obfuscate scripts is ok, but pack obfuscated scripts with PyInstaller got error.

Sample code to help reproduce the issue:

1. Create `a.py` with one line: `print('123')`
2. Call `pyarmor pack a.py`
3. `cd dist/a` then `./a` got following:

```bash
Traceback (most recent call last):
  File "projects/pyinstaller/dist/a.py", line 1, in <module>
NameError: name '__pyarmor__' is not defined
[99992] Failed to execute script a
```

---

Package version:
- pyarmar == 5.2.8
- pyinstaller == 3.4

---

Did I miss something?

**Comment by jondy** [comment]
There is a bug in this version, I'm fixing it.

**Comment by jondy** [comment]
Fix it in v5.2.9, please upgrade PyArmor to the latest version.
